linux/debian/10/foxpass_setup.py

```diff
@@ -0,0 +1,258 @@
+#!/usr/bin/python
```

Comment:
Should we specify python3, or is that the default on debian10?

Reply:
We should specify python3

Reply:
I made it python3 by default.